src/megachatapi_impl.cpp

```diff
@@ -513,13 +512,25 @@ void MegaChatApiImpl::sendPendingRequests()
 
             if (!chatroom->isGroup()
                 || !chatroom->publicChat()
-                || !chatroom->previewMode())   // Join only for public group chats in preview mode
+                || (!chatroom->previewMode() && chatroom->isActive()))   // Join only for public group chats in preview mode
```

Comment:
In preview mode or not in preview mode but inactive (the comment above needs to be updated)

Reply:
I think that It could be updated by: Cannot autojoin an active chat if it's not in preview mode.
What do you think?